### ISSUE TITLE
metallicRoughnessTexture encoding is wrong

### ISSUE THREAD

**Issue by helviett** [KhronosGroup/glTF-Sample-Models#398]
The spec [says](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#metallic-roughness-material):

> The textures for metalness and roughness properties are packed together in a single texture called metallicRoughnessTexture. Its green channel contains roughness values and its blue channel contains metalness values. This texture *MUST* be encoded with linear transfer function and *MAY* use more than 8 bits per channel.

But metallic-roughness textures in Sponza folder are encoded with sRGB. Checked it with ImageMagick and PVRTexTool.

**Comment by javagl** [comment]
Depending on whether someone can definitely say whether this is duplicate of the issue around https://github.com/KhronosGroup/glTF-Sample-Models/issues/316#issuecomment-860333473 , it might be closed. (I think it is, but others should confirm...)

**Comment by donmccurdy** [comment]
**tl;dr** – the ICC profiles are often wrong, the data is likely _not_ sRGB. Agreed it'd be nice if we can fix the ICC profiles, that's a bit of a messy area in photo-editing software historically IMHO. Implementations of glTF are required to ignore the profiles for that reason.

I'd be open to putting something in glTF-Transform to fix incorrect ICC profiles too though, it'd be ideal to have these sample be clean unless there're explicitly testing that requirement. The library I'm currently using (sharp.js) does not let me edit the ICC profile without re-encoding the image, unfortunately.

**Comment by DrX3D** [comment]
This repo is being ARCHIVED. Transferred to [glTF-Sample-Assets](https://github.com/KhronosGroup/glTF-Sample-Assets).
